examples/plot_discriminant_scores.py

```diff
@@ -81,12 +43,41 @@
 )
 
 # Add the histograms
-plot_histo.add(hist_dips_light, reference=True)
-plot_histo.add(hist_dips_c, reference=True)
-plot_histo.add(hist_dips_b, reference=True)
-plot_histo.add(hist_rnnip_light)
-plot_histo.add(hist_rnnip_c)
-plot_histo.add(hist_rnnip_b)
+for tagger, linestyle in zip(taggers, linestyles):
+
+    plot_histo.add(
+        Histogram(
+            df[is_light][f"disc_{tagger}"],
+            label="Light-flavour jets" if tagger == "dips" else None,
```

Comment:
can you add comments why you are doing that?

Reply:
done